src/lib/data/banners/events/3.5.json

```diff
@@ -69,6 +69,71 @@
 					}
 				}
 			}
+		},
+		{
+			"phase": 2,
+			"banners": {
+				"standardVersion": 3,
+				"events": {
+					"item": [
+						{
+							"name": "the-transcendent-one-returns-2",
+							"character": "shenhe",
+							"buttonBoxPosition": {
+								"w": 160,
+								"t": 22
+							},
+							"textOffset": {
+								"t": 78.5,
+								"l": 43
+							}
+						},
+						{
+							"name": "the-heron_s-court-3",
+						"character": "kamisato-ayaka",
+							"buttonBoxPosition": {
+								"w": 95,
+								"t": 32,
+								"l": 50
+							},
+							"textOffset": {
+								"l": 51
+							}
+						}
+					],
+
+					"rateup": ["diona", "sucrose", "mika"]
+				},
+				"weapons": {
+					"name": "epitome-invocation-44",
+					"fatepointsystem": true,
+					"featured": [
+						{
+							"name": "calamity-queller",
+							"type": "polearm",
+							"buttonBoxPosition": {
+								"t": 15,
+								"w": 90,
+								"l": 60
+							}
+						},
+						{
+							"name": "mistsplitter-reforged",
+							"type": "sword",
+							"buttonBoxPosition": {
+								"t": 5,
+								"w": 80,
+								"l": 35
+							}
+						}
```

Comment:
The Weapon button as well, you can add more number for calamity to set the icon on the right side.

https://github.com/AguzzTN54/Genshin-Impact-Wish-Simulator/blob/36fe7a881a311dd75cd37df1ffff6e7c191becc9/src/lib/data/banners/events/2.4.json#L44-L50

https://github.com/AguzzTN54/Genshin-Impact-Wish-Simulator/blob/36fe7a881a311dd75cd37df1ffff6e7c191becc9/src/lib/data/banners/events/2.0.json#L28-L34